### DISCUSSION TITLE
One validation message for more than one component (but not one summary)

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#3143] [Q&A]
Can I get a combined feedback message, to the setup below?

I have these inputs
![image](https://user-images.githubusercontent.com/2386572/142613881-931ae1ca-bb31-4310-8412-9c7fa8997554.png)

And only 1 of these inputs can be 0 but **not** both of them.

like this
hours: 0 and minutes: 1
hours: 1 and minutes: 0
but not hours: 0 and minutes: 0

```xml
<Row>
     <Validation Validator="@ValidateEventLenght">
        <Column ColumnSize="ColumnSize.Is4.OnDesktop">
            @L["Hours"] <NumericEdit Value="0" Max=24 @bind-Text="@hours"/>
        </Column>
        <Column ColumnSize="ColumnSize.Is4.OnDesktop">
            @L["Minutes"]  <NumericEdit Value="0" Max=59 @bind-Text="@minutes"/>
        </Column>
    </Validation>
</Row>
```

```charp
int minutes =0;
int hours=0;

private void ValidateEventLenght(ValidatorEventArgs e)
{
    if(hours == 0 && minutes ==0)
    {
        e.Status = ValidationStatus.Error;
        e.ErrorText = localizer["YouNeedToSpecifyTheLengthOfTheEvent"];
    }
}
```

and I can´t get the message to fit in 

```xml            
<Feedback>
    <ValidationError></ValidationError>
</Feedback>
```

**Comment by stsrki** [comment]
Look at the ValidationSummary in our docs: https://blazorise.com/docs/components/validation. It is used to combine errors.

And for validation of multiple fields, you would probably have to implement your own rules. That each references the other field somehow.
